src/app/features/charts/components/median-barchart/median-barchart.component.ts

```diff
@@ -0,0 +1,260 @@
+        .range([0, innerWidth])
+        .padding(0.2);
+
+      const yScale = d3
+        .scaleLinear()
+        .domain([0, this.maxValueY])
+        .nice()
+        .range([innerHeight, 0]);
+
+      // BARS
+      this.chart
+        .selectAll('.medianbars')
+        .data(this._data)
+        .enter()
+        .append('rect')
+        .attr('class', 'medianbars')
+        .attr('x', (d) => xScale(d.tissue) as number)
+        .attr('y', (d) => yScale(d.medianlogcpm || 0))
+        .attr('width', xScale.bandwidth())
+        .attr('height', (d) => innerHeight - yScale(d.medianlogcpm || 0))
+        .attr('fill', barColor);
+
+      // SCORE LABELS
+      this.chart
+        .selectAll('.bar-labels')
+        .data(this._data)
+        .enter()
+        .append('text')
+        .attr('class', 'bar-labels')
+        .attr('x', (d) => this.getBarCenterX(d.tissue, xScale))
+        .attr('y', (d) => yScale(d.medianlogcpm || 0) - 5)
+        .text((d) => this.helperService.roundNumber(d.medianlogcpm || 0, 2));
+
+      // X-AXIS
+      const xAxis = d3.axisBottom(xScale);
+      this.chart
+        .append('g')
+        .attr('class', 'x-axis')
+        .attr('transform', `translate(0, ${innerHeight})`)
+        .call(xAxis.tickSizeOuter(0))
+        .selectAll('.tick')
+        .on('mouseenter', (_, tissue) => {
+          const tooltipText = this.helperService.getGCTColumnTooltipText(
+            tissue as string
+          );
+          this.showTooltip(
+            tooltipText,
+            this.getBarCenterX(tissue as string, xScale) + margin.left,
+            height - margin.top
+          );
+        })
+        .on('mouseleave', () => {
+          this.hideTooltip();
+        });
+
+      // Y-AXIS
+      const yAxis = d3.axisLeft(yScale);
+      this.chart.append('g').attr('class', 'y-axis').call(yAxis);
+
+      // X-AXIS LABEL
+      this.chart
+        .append('text')
+        .attr('class', 'x-axis-label')
+        .attr('x', innerWidth / 2)
+        .attr('y', innerHeight + margin.bottom)
+        .attr('text-anchor', 'middle')
+        .text(this.xAxisLabel);
+
+      // Y-AXIS LABEL
+      this.chart
+        .append('text')
+        .attr('class', 'y-axis-label')
+        .attr('x', -innerHeight / 2)
+        .attr('y', -margin.left)
+        .attr('dy', '1em')
+        .attr('text-anchor', 'middle')
+        .attr('transform', 'rotate(-90)')
+        .text(this.yAxisLabel);
+
+      // THRESHOLD LINE
+      this.chart
+        .append('line')
+        .attr('class', 'meaningful-expression-threshold-line')
+        .attr('x1', 0)
+        .attr('x2', innerWidth)
+        .attr('y1', yScale(this.MEANINGFUL_EXPRESSION_THRESHOLD))
+        .attr('y2', yScale(this.MEANINGFUL_EXPRESSION_THRESHOLD))
+        .attr('stroke', 'red');
+
+      this.chartInitialized = true;
+    }
+  }
+
+  @HostListener('window:resize', ['$event'])
+  onResize() {
+    this.clearChart();
+    this.createChart();
+  }
```

Comment:
Handles resizing the chart when the window is resized -- 


https://github.com/Sage-Bionetworks/Agora/assets/26949006/3652a553-2c7c-4898-ac83-ef9b09b43eb3

Reply:
nice

Reply:
Updated to use transitions when chart is resized on window resize to match the score-barchart transition: 

https://github.com/Sage-Bionetworks/Agora/assets/26949006/1d97769b-58e5-4676-84c5-2c82fddc022f